packages/mongodb-memory-server-core/src/util/__tests__/MongoBinary.test.ts

```diff
@@ -138,6 +138,32 @@ build environment:
         expect(output).toBe(sysBinaryPath);
       });
 
+      it('should return and check an SystemBinary and match even if binary has patch version', async () => {
+        // Output taken from mongodb x64 for "ubuntu" version "4.0.25"
+        // DO NOT INDENT THE TEXT
+        jest.spyOn(childProcess, 'spawnSync').mockReturnValue(
+          // @ts-expect-error Because "Buffer" is missing values from type, but they are not used in code, so its fine
+          {
+            stdout: Buffer.from(`db version v4.2.19-11-ge2f2736
+git version: e2f27369cdfb8c417b026afb323c810226417206
+OpenSSL version: OpenSSL 1.1.1n  15 Mar 2022
+allocator: tcmalloc
+modules: none
+build environment:
+    distmod: debian10
+    distarch: x86_64
+    target_arch: x86_64`),
+          }
+        );
+        process.env[envName(ResolveConfigVariables.VERSION)] = '4.2.19'; // set it explicitly to that version to test matching versions
```

Comment:
i have not seen the test result yet, but is it intended that only the 2 dot version is meant to be there and not also what is after the `-` too?

Reply:
That's probably up for debate, but I'd argue that matching the major, minor, and patch version is a reasonable approach, though perhaps it should also match the specific version (with tags)

Reply:
(so yeah that was my intent, but I'm open to someone suggesting it should behave otherwise)